src/backend/metal/Cargo.toml

```diff
@@ -33,4 +33,4 @@ libc = "0.2"
 objc = "0.1.8"
 objc-foundation = "0.1"
 bit-set = "0.4"
-metal-rs = "0.1"
+metal-rs = { git = "https://github.com/fkaa/metal-rs", rev = "5263898c2ce1fb8263653272014366eba45ef88d" }
```

Comment:
Why do you need a git revision here?
We may just ask @fkaa to publish a new version instead.
Although, I don't mind patching this later on.